con-master/src/main/java/ba/codecentric/master/controller/FacilityTypeController.java

```diff
@@ -0,0 +1,26 @@
+package ba.codecentric.master.controller;
+
+import ba.codecentric.base.domain.FacilityType;
+import ba.codecentric.base.service.FacilityTypeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class FacilityTypeController {
+
+    private FacilityTypeService facilityTypeService;
+
+    @Autowired
+    public FacilityTypeController(FacilityTypeService facilityTypeService) {
+        this.facilityTypeService = facilityTypeService;
+    }
+
+    @RequestMapping(value = "/facilities/types", method = RequestMethod.GET)
```

Comment:
Misleading URL name or controller. Either move this method to `FacilityController` or rename URL to something like "facility-types".